pyomo/contrib/mindtpy/config_options.py

```diff
@@ -6,7 +6,7 @@
 def _get_GDPopt_config():
     CONFIG = ConfigBlock("MindtPy")
     CONFIG.declare("bound_tolerance", ConfigValue(
-        default=1E-6,
+        default=1E-4,
```

Comment:
Just to verify, tests are passing here, right?

Reply:
Yep